packages/client/lib/config.ts

```diff
@@ -282,4 +282,11 @@ export class Config {
     if (option !== undefined) return option
     return this.chainCommon.chainName() === 'mainnet'
   }
+
+  getNetworkDir(): string {
+    const networkDirName = this.common.chainName()
+    const dataDir = `${this.datadir}/${networkDirName}`
+
+    return dataDir
```

Comment:
Please use the same naming scheme here as in the other directory methods - so `getNetworkDirectory()` and sort this in above along with the other methods, a bit misplaced here.

Reply:
Can you also use this method for directory creation in both of the other two methods?